br/pkg/task/stream.go

```diff
@@ -922,7 +922,7 @@ func RunStreamTruncate(c context.Context, g glue.Glue, cmdName string, cfg *Stre
 	removed := metas.RemoveDataBefore(shiftUntilTS)
 
 	// remove metadata
-	removeMetaDone := console.ShowTask("Removing metadata... ", glue.WithTimeCost())
+	removeMetaDone := console.ShowTask("Removing Metadata... ", glue.WithTimeCost())
 	if !cfg.DryRun {
 		if err := metas.DoWriteBack(ctx, storage); err != nil {
```

Comment:
Currently, the log file is removed first, and then the meta file is removed or written back. Shall we put `metas.DoWriteBack` into Progress? And call `p.Wait()` after `removeMetaDone()`.
e.g.,
```
metas.DoWriteBack(ctx, storage, func() {
    p.Inc()
})
```